about-style.css

```diff
@@ -0,0 +1,31 @@
+.photo {
+  width: 620px;
+  height: 300px;
+  border: 2px solid #252525;
+  background-image: url(./images/g8.jpg);
```

Comment:
you'll want to wrap the path in quotes. it works. but the correct syntax is ` url(./images/g8.jpg);`